src/app/static/src/js/components/runReportTabs/runningReportsDetails.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+    <div>
+        <div class="container">
+            <div id="report-log">
+                <div class="row pt-2">
+                    <div id="report-git-branch" v-if="reportLog.git_branch" class="col-sm-auto">
+                        <div class="text-right">
+                            <span>Github branch:</span>
```

Comment:
I know you've followed the mockups here, but for consistency with report running page (and planned workflow running page) I think this should be "Git" rather than "Github" - there is no dependency between orderly and GitHub